contracts/src/7683/T1ERC7683.sol

```diff
@@ -127,7 +155,27 @@ contract T1ERC7683 is BasicSwap7683, OwnableUpgradeable, PausableUpgradeable {
     /// @param destinationDomain The domain of the destination chain
     /// @param orderId The ID of the order to verify
     /// @return requestId The ID of the read request
-    function verifySettlement(uint32 destinationDomain, bytes32 orderId) external returns (bytes32 requestId) {
+    function verifySettlement(uint32 destinationDomain, bytes32 orderId) external payable returns (bytes32 requestId) {
```

Comment:
Good call on removing the `destinationDomain` param on `verifyRefund`. perhaps we should also use destination domain from `orderData` instead of passing it in?

Reply:
I thought of it but the increase in gas would be big here as we don't already get orderData like in verifyRefund.
Do you think it's still worth it ?